### PR TITLE
Add spec draft

### PR DESCRIPTION
**Pull request by phoglenix**
***
<a href="https://pr-preview.s3.amazonaws.com/phoglenix/digital-goods/pull/40.html" title="Last updated on Feb 2, 2022, 12:16 AM UTC (baac301)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/digital-goods/40/7bd9e5b...phoglenix:baac301.html" title="Last updated on Feb 2, 2022, 12:16 AM UTC (baac301)">Diff</a>

### CODE REVIEW

File: spec.bs [hunk 1]
```diff
@@ -5,13 +5,419 @@ Level: 1
...
+    new PaymentRequest(
+      [{supportedMethods: 'https://example.com/billing',
+        data: {itemId: item.itemId}}]);
+  </xmp>
+
+  The purchase flow itself uses the [[payment-request|Payment Request API]]. We
+  don’t show the
+  full payment request code here, but note that the item ID for any items the
+  user chooses to purchase can be sent in the <code>data</code> field of a
+  <code>modifiers</code> entry
```
Comment (rsolomakhin) [inline spec.bs:107]:
```suggestion
  <code>methodData</code> entry
```

**Review by rsolomakhin — COMMENTED** [review]
Looks good % a couple of small corrections.

File: spec.bs [hunk 2]
```diff
@@ -5,13 +5,419 @@ Level: 1
...
+      src="https://sub.origin.example"
+      allow="payment">
+    </iframe>
+  </xmp>
+
+  To indicate that a subdomain iframe is allowed to invoke the Digital Goods API,
+  the <code>allow</code> attribute along with the <code>"payment"</code> keyword
+  can be specified on the
+  iframe element. Cross-origin iframes cannot invoke the Digital Goods API. The
+  [[Permissions Policy]] specification provides further details and examples.
```
[the remark this answers is not in the document]
Comment (phoglenix) [reply spec.bs]: Done

**Review by rsolomakhin — APPROVED** [review]
LGTM

**Pull request merged**